client/my-sites/plan-features/index.jsx

```diff
@@ -50,37 +51,63 @@ import { retargetViewPlans } from 'lib/analytics/ad-tracking';
 import { abtest } from 'lib/abtest';
 
 class PlanFeatures extends Component {
-
 	render() {
-		const { planProperties } = this.props;
-
+		const {
+			planProperties,
+			isInSignupTest
+		} = this.props;
 		const tableClasses = classNames( 'plan-features__table',
 			`has-${ planProperties.length }-cols` );
+		const planClasses = classNames(
+			'plan-features',
+			{ 'plan-features--signup': isInSignupTest }
+		);
+		const planWrapperClasses = classNames(
+			{ 'plans-wrapper': isInSignupTest }
+		);
+		let mobileView, planDescriptions;
+		let bottomButtons = null;
+
+		if ( ! isInSignupTest ) {
+			mobileView = (
+				<div className="plan-features__mobile">
+					{ this.renderMobileView() }
+				</div>
+			);
+
+			planDescriptions = (
+				<tr>
+					{ this.renderPlanDescriptions() }
+				</tr>
+			);
+
+			bottomButtons = (
+				<tr>
+					{ this.renderBottomButtons() }
+				</tr>
+			);
+		}
 
 		return (
-			<div>
-				{ this.renderUpgradeDisabledNotice() }
-				<div className="plan-features__content">
-					<div className="plan-features__mobile">
-						{ this.renderMobileView() }
-					</div>
-					<table className={ tableClasses }>
-						<tbody>
-							<tr>
-								{ this.renderPlanHeaders() }
-							</tr>
-							<tr>
-								{ this.renderPlanDescriptions() }
-							</tr>
-							<tr>
-								{ this.renderTopButtons() }
-							</tr>
+			<div className={ planWrapperClasses } ref="plansWrapper">
```

Comment:
According to the React official documentation, **string refs are likely to be removed in one of the future releases.** You should use function callbacks instead.
https://facebook.github.io/react/docs/refs-and-the-dom.html

This would be helpful: https://github.com/Automattic/wp-calypso/blob/master/client/components/tinymce/plugins/mentions/mentions.jsx#L243